docs/src/features.md

```diff
@@ -117,7 +80,7 @@ You can change the default keybinding by remapping the command as below:
 ```js
 {
     "command": "auto_complete_open_link",
-    "keys": ["f12"],
+    "keys": ["f13"],
```

Comment:
Since this example is about overriding the default F12 shortcut, I've changed it to something else than F12.

Reply:
Maybe this example isn't really necessary at all, because it is just a default keybinding from ST and not specific to LSP? And you have already described how to define/override a keybinding in customization.md.
Btw, I think F-keys larger than F12 are macOS exclusive.

Reply:
Yeah, I agree, removed.